clients/java/src/main/java/io/camunda/zeebe/client/ZeebeClientBuilder.java

```diff
@@ -118,6 +118,12 @@ ZeebeClientBuilder applyEnvironmentVariableOverrides(
    */
   ZeebeClientBuilder overrideAuthority(String authority);
 
+  /**
+   * A custom zeebe client maxMessageSize which will be used to apply grpc client maxInboundMessageSize.
+   * The default is 4194304 = 4MB.
+   */
+  ZeebeClientBuilder maxMessageSize(int maxSize);
```

Comment:
🔧 This documentation is quite technical. We could consider adding what a user gains by setting it. For example, something like 

> A custom maxMessageSize allows the client to receive larger or smaller responses from Zeebe. Technically, it specifies the maxInboundMessageSize of the gRPC channel. The default is 4194304 = 4MB.

Reply:
fixed